misc/pause-container/Makefile

```diff
@@ -20,7 +20,7 @@ TAG = 0.1.0
 all: container
 
 container:
-	docker build . -t "$(IMAGE):$(TAG)"
```

Comment:
It doesn't work on old docker version, to be specific 1.11.2. Let's make it more compatible.